google/ggrc-core
# GGRC-5878 Remove instance wrapper for modal content to avoid unneeded re-render

# Issue description

There are modal_content.mustache templates (are used to render "Create/Edit" modals), where the content is wrapped in {{#instance }}{{/instance }} construction. When the instance is changed, then inner content will be re-rendered (we can change "instance" via "Save & Add another" button). It leads to rebuilding of inner components + repainting of static data (for example, some text, buttons ...).

# Steps to test the changes

1. Open New Program/Audit/etc. modal.
2. Fill modal fields.
3. Click Save&Add Another button.
4. Verify that modal content wasn't fully re-rendered, only fields values. 

# Solution description

For Workflow and Task Group/Task modal content instance wrapper was removed before. The same can be performed for other objects modals, as content components can use instance value directly. 
1. Removed instance wrapper construction.
2. Updated components to use instance value directly. 
# Sanity checklist




# PR Review checklist
